src/components/intercept/config/electron-config.tsx

```diff
@@ -121,6 +121,12 @@ function getReadablePath(path: string) {
     }
 }
 
+declare global {
+    interface Window {
+        nativeFile: { open: () => Promise<string | undefined> };
```

Comment:
This type isn't correct - the app can run in a browser without Electron, or could run in an older version of the desktop shell, where `nativeFile` won't exist. That should be `nativeFile?` I think.

Reply:
Ah, yes, `nativeFile?` is correct.